README.md

```
@@ -0,0 +1,142 @@
+
+## Usage
+
+```sh
+aur-manager scan <path>
+```
+
+`<path>` is either a `PKGBUILD` file or a directory containing one (an AUR git
+checkout, typically) — any `*.install`, `*.patch`, or `*.diff` files next to it are
+picked up automatically.
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)